src/testing/AddTestToIdentifierPage.test.tsx

```diff
@@ -14,77 +15,140 @@ jest.mock('../common/useConfig');
 describe(AddTestToIdentifierPage, () => {
   beforeEach(() => {
     mockConfigForEstonianIdMethodAndEnglish();
-    mockToken('some-token');
+    mockAuthentication();
   });
 
   it('focuses identifier input', async () => {
-    const { getByLabelText } = renderWrapped(<AddTestToIdentifierPage />);
+    mockHttp().get('/api/v1/test-types').reply(200, [aTestType()]);
 
-    const identifierInput = getByLabelText(/identification code/i);
+    renderWrapped(<AddTestToIdentifierPage />);
 
+    const identifierInput = screen.getByLabelText(/identification code/i);
     expect(identifierInput).toHaveFocus();
   });
 
-  it('creates a user for the authentication method from config and the filled identifier', async () => {
-    const authenticationDetails = { method: 'ESTONIAN_ID', identifier: '39210030814' };
-    const scope = nock(/./)
-      .post('/api/v1/users', { authenticationDetails })
-      .matchHeader('Authorization', 'Bearer some-token')
-      .reply(201, { id: 'some-id', authenticationDetails });
+  it('creates a user for the authentication method from config and the filled identifier and adds a test for that user', async () => {
+    mockHttp().get('/api/v1/test-types').reply(200, [aTestType()]);
 
-    const { getByText, getByLabelText, queryByText, findByText } = renderWrapped(
-      <AddTestToIdentifierPage />
-    );
+    renderWrapped(<AddTestToIdentifierPage />);
 
-    const identifierInput = getByLabelText(/identification code/i) as HTMLInputElement;
-    const submit = () => fireEvent.click(getByText(/add test result/i));
+    const identifierInput = screen.getByLabelText(/identification code/i);
+    const button = screen.getByRole('button');
```

Comment:
I assume this is the submitButton, could you rename?  I found it unclear further down the test.

Reply:
✅ Renamed!